packages/realm/RealmJS.podspec

```diff
@@ -35,36 +37,66 @@ Pod::Spec.new do |s|
+  # The source field is a required field in the podspec, but it is not meant to be used.
+  # This is because the Podspec is not meant to be published into a CocoaPod repository, instead React Native uses a :path style dependency when adding this to the users projects Podfile.
   # @see https://guides.cocoapods.org/using/the-podfile.html#using-the-files-from-a-folder-local-to-the-machine
   # @see https://github.com/react-native-community/cli/blob/master/docs/autolinking.md#platform-ios
   s.source                 = { :http => 'https://github.com/realm/realm-js/blob/main/CONTRIBUTING.md#how-to-debug-react-native-podspec' }
 
-  s.source_files           = 'react-native/ios/RealmReact/*.mm'
-  s.public_header_files    = 'react-native/ios/RealmReact/*.h'
+  s.source_files           = 'react-native/ios/RealmReact/*.mm',
+                             'react-native/shared/*.cpp',
+                             'binding/ios/platform.mm'
+
+  s.public_header_files    = 'react-native/ios/RealmReact/*.h',
+                             'react-native/ios/build/include/**/*.{h,hpp}'
 
   s.frameworks             = uses_frameworks ? ['React'] : []
 
   s.library                = 'c++', 'z', 'compression'
 
   s.pod_target_xcconfig    = {
                                 # Setting up clang
-                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17',
+                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++20',
                                 'CLANG_CXX_LIBRARY' => 'libc++',
                                 # Setting the current project version and versioning system to get a symbol for analytics
                                 'CURRENT_PROJECT_VERSION' => s.version,
                                 'VERSIONING_SYSTEM' => 'apple-generic',
+                                'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) REALM_ENABLE_SYNC=1',
+                                'GCC_SYMBOLS_PRIVATE_EXTERN' => 'YES',
                                 # Header search paths are prefixes to the path specified in #include macros
                                 'HEADER_SEARCH_PATHS' => [
                                   '"$(PODS_TARGET_SRCROOT)/react-native/ios/RealmReact/"',
-                                  '"$(PODS_ROOT)/Headers/Public/React-Core/"'
-                                  #"'#{app_path}/ios/Pods/Headers/Public/React-Core'" # Use this line instead of 👆 while linting
+                                  '"$(PODS_TARGET_SRCROOT)/react-native/ios/build/include/"',
+                                  '"$(PODS_TARGET_SRCROOT)/binding/"',
+                                  '"$(PODS_TARGET_SRCROOT)/bindgen/src/"',
+                                  '"$(PODS_TARGET_SRCROOT)/bindgen/vendor/realm-core/bindgen/src/"'
                                 ].join(' ')
                               }
+  # Create placeholders for vendored_libraries, so they are added to the xcode project
+  s.prepare_command = <<-EOS
```

Comment:
Instead of using a prepare command, why not simply execute the scripts right here and now?

Reply:
I don't understand why not do it in the prepare command.  I want this to run exactly in the phase described [here](https://guides.cocoapods.org/syntax/podspec.html#prepare_command)

Reply:
Oh. Re-reading this, I confused this with a [script_phases](https://guides.cocoapods.org/syntax/podspec.html#script_phases). I agree this seems to be serving the purpose 👍